core/runtime/bindings.rs

```diff
@@ -179,17 +180,26 @@ pub(crate) fn initialize_context<'s>(
 
     // Bind v8 console object to Deno.core.console
     let extra_binding_obj = context.get_extras_binding_object(scope);
-    let console_obj: v8::Local<v8::Object> = get(
-      scope,
-      extra_binding_obj,
-      b"console",
-      "ExtrasBindingObject.console",
-    );
+    let console_obj: v8::Local<v8::Object> =
+      get(scope, extra_binding_obj, b"console", "%console");
+    let is_one_byte_fn: v8::Local<v8::Object> =
+      get(scope, extra_binding_obj, b"isOneByte", "%isOneByte");
+    let from_utf8_fn: v8::Local<v8::Object> =
+      get(scope, extra_binding_obj, b"fromUtf8", "%fromUtf8");
+    let to_utf8_fn: v8::Local<v8::Object> =
+      get(scope, extra_binding_obj, b"toUtf8", "%toUtf8");
```

Comment:
I wouldn't worry about it for this patch, but I think we could just pass the entire extra_binding_obj into the function and allow the function to extract these various fields.

Reply:
I agree, I spoke with @littledivy about this today and the whole `Deno.__op__ = ...` bit seems like something that should be looked at again and cleaned up.

Reply:
Agreed...  op binding is pretty rough.